src/m365/purview/commands/retentioneventtype/retentioneventtype-add.spec.ts

```diff
@@ -0,0 +1,123 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { telemetry } from '../../../../telemetry';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+import { accessToken } from '../../../../utils/accessToken';
+const command: Command = require('./retentioneventtype-add');
+
+describe(commands.RETENTIONEVENTTYPE_ADD, () => {
+  const displayName = 'Contract Expiry';
+  const description = 'A retention event type description';
+
+  const requestResponse = {
```

Comment:
Shall we add this in a //# Mock Responses section? Check out how some other commands do this